mhkit/power/quality/flicker_assessment_workflow.m

```diff
@@ -0,0 +1,100 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
I love the idea of having a workflow function tying all the flicker calculations together. Please restructure to take inputs and have MHKiT style docstring

Reply:
Thanks! I merged this code into the power_example, please check it out and see if we should delete this one.